src/components/Table/Table.tsx

```diff
@@ -395,7 +395,7 @@ class Table extends React.Component<Props, State> {
             })
           }
 
-          { rowAction && !rowActionLeft ? <TableData componentClass={theme.lastData} componentStyle={{ float: 'right' }}>{item.isRowLoading && <Spinner componentSize="small" componentColor="disabled" />} <RowAction componentId={componentId} actionInProgress={actionInProgress && !!item.processing} isRowLoading={item.isRowLoading} actionConfig={rowAction} data={item} theme={theme} /> </TableData> : '' }
+          { rowAction && !rowActionLeft ? <TableData componentClass={theme.lastData} componentStyle={{ float: 'right', width: '72px', textAlign: 'center' }}>{item.isRowLoading && <Spinner componentSize="small" componentColor="disabled" />} <RowAction componentId={componentId} actionInProgress={actionInProgress && !!item.processing} isRowLoading={item.isRowLoading} actionConfig={rowAction} data={item} theme={theme} /> </TableData> : '' }
```

Comment:
Is float: 'right' required? Shouldn't the table automatically fill the space as long as 1 column has width auto change made in https://github.com/emgage/emgage-frontend/pull/1733/files

Reply:
No there is no requirement of float. because we are passing width and text-align center. i will remove the float property from it